source/group_eq_bot/interfaces/telegram_event_handlers/conversations/validation/handler.py

```diff
@@ -0,0 +1,189 @@
+from dataclasses import dataclass
+from typing import Union
+
+from hydra import compose, initialize
+from hydra.core.global_hydra import GlobalHydra
+
+from telegram import Update as TelegramEvent
+from telegram.ext import ConversationHandler, CommandHandler, MessageHandler, filters, ContextTypes
+
+from utilities.internal_logger.logger import logger
+from interfaces.models.conversation_handler_states import ConversationStates
+from interfaces.telegram_event_handlers.conversations.validation.helpers import ConversationValidatorHelpers
+
+# Fetch bot configuration with hydra compose api
+# https://hydra.cc/docs/advanced/compose_api/
+initialize(version_base="1.2", config_path="../../../../configurations", job_name="conversation_validation_handler")
+configurations = compose(config_name="configuration")
+GlobalHydra.instance().clear()
+
+LIMIT_OF_ATTEMPTS_TO_FAIL_PER_QUESTION: int = int(configurations.validation.limit_of_attempts_to_fail_per_question)
+MATHEMATICAL_ANSWER: str = configurations.validation.mathematical_answer
```

Comment:
Based on the general PR comment #3 I left, please review the structure of the file as well as the names of the classes.

Detailed recommendation:
```
reference to the folder structure:
- conversation_update
            - helpers
                - ...
            - states
                - text
                    - state.py
                - audio
                    - state.py
            - handler.py
```

1. `handler.py` has to hold JUST the logic for constructing the conversation states
```python
Conversation = ConversationHandler(entry_points=[], states={}, fallbacks=[])
```

2. `states.text.state.py` has to hold the interface to form the text question state
3. `states.audio.state.py` has to hold the interface to form the audio question state